src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,137 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  inputHandler = (event) => {
```

Comment:
```suggestion
  handleChange = (event) => {
```

it's a function, name it with verbs

Reply:
+